.github/workflows/publish.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
 
       - name: publish
         run: |
-          pnpm config set -H 'npmAuthToken' "${{secrets.NPM_TOKEN}}"
-          pnpm npm publish --access public --tolerate-republish
+          pnpm config set npmAuthToken "${{secrets.NPM_TOKEN}}"
+          pnpm publish --force
```

Comment:
Why is `--force` needed here?

Reply:
So, we can republish 😄 , basically equal to `--tolerate-republish`

Reply:
Ok, but when would we need to republish? Is a different part of CI pushing a version number, and this comes up later and pushes the code? 🤔

Reply:
Nope, I think its there as a safeguard to remove certain stuff from a release if needed and you can't publish a new version for it. I copied it over from backstage itself but we can remove it if you want

Reply:
[I found the PR where Backstage added `--tolerate-republish`](https://github.com/backstage/backstage/pull/13848) - maybe it's for re-publishing a _tag_ (so they can keep updating what `@nightly` refers to?)

Either way, you're right that `--force` here is like `--tolerate-republish`, and I missed that you meant that as in "hey, we had it before, so it's more of a behaviour change to _remove_ it".

So, 👍 either way, whether you want to keep it or remove it.